autoload/neomake.vim

```diff
@@ -528,17 +530,16 @@ function! neomake#GetEnabledMakers(...) abort
     return enabled_makers
 endfunction
 
-function! s:HandleLoclistQflistDisplay(file_mode) abort
-    let open_val = get(g:, 'neomake_open_list')
-    if open_val
+function! s:HandleLoclistQflistDisplay(open_val, file_mode) abort
```

Comment:
Feel free to skip it then: I will pick this up after the open PRs about tempfiles, autolinting etc are finished/merged.